src/diffusers/models/modeling_utils.py

```diff
@@ -458,18 +458,35 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 " dispatching. Please make sure to set `low_cpu_mem_usage=True`."
             )
 
+        # Load config if we don't provide a configuration
+        config_path = pretrained_model_name_or_path
+
         user_agent = {
             "diffusers": __version__,
             "file_type": "model",
             "framework": "pytorch",
         }
 
-        # Load config if we don't provide a configuration
-        config_path = pretrained_model_name_or_path
+        # load config
+        config, unused_kwargs = cls.load_config(
```

Comment:
So this is the only GET request that we should perform, is that correct? I suppose there's no way to replace it with a HEAD to retrieve the hash if a config is already cached, is it?

Reply:
Actually that's a HEAD request, but yeah as you said below it's not enough